react-app/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You haven't been using much of TypeScript in your code, though you seem to be having necessary dependencies installed and basic `tsconfig.json` set up. I would recommend to switch your components from `.js` to `.tsx` and play around with actual TypeScript in your components to fully take advantages of TS benefits.